public/data/processes.json

```diff
@@ -6805,7 +6775,7 @@
   },
   {
     "name": "Vapeur à partir de gaz naturel (mix de technologies de combustion et d'épuration des effluents gazeux|en sortie de chaudière|Puissance non spécifiée), RER",
-    "info": "Energie > Chaleur > Mix moyen",
+    "info": "Energie > Chaleur > Vapeur par énergie primaire",
```

Comment:
Note to reviewers, esp. incl. @paulboosz: I also took the liberty to categorize this process the exact same way similar ones are.